leakcanary-android-leak-deobfuscation-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'maven'
+
+gradlePlugin {
+  plugins {
+    leakCanary {
+      id = 'com.squareup.leakcanary.leak.deobfuscation.plugin'
+      implementationClass =
+          'com.squareup.leakcanary.leak.deobfuscation.plugin.LeakcanaryLeakDeobfuscationPlugin'
+    }
+  }
+}
+
+dependencies {
+  implementation deps.kotlin.stdlib
+  implementation deps.kotlin.gradlePlugin
+  implementation deps.androidGradlePlugin
+  compileOnly gradleApi()
```

Comment:
@mzgreen you should add `testImplementation` dependencies for all libs used in tests (junit, assertj, etc)

Reply:
```
testImplementation deps.junit
testImplementation deps.mockito
testImplementation deps.mockito_kotlin
testImplementation deps.assertj_core
```

Reply:
@mzgreen when you are coding in AS, do not let the IDE add the dependencies for you. It adds them to its own configuration -> it will work locally but not for anybody else. You need to manage your dependencies from gradle.

I strongly suggest you get more comfortable with gradle and how to build your project from the command line. Here I guess you were missing to run the unit tests, you were only running the android instrumented/connected tests. Also, the `build` target is not well defined on Android, the simplest would probably have been to run `./gradlew check`.

Reply:
@stephanenicolas It may seem like I don't know what I'm doing and I'm no gradle expert but I'm not a total beginner either 😃  Thanks to your response I figured it out. I had maven local publishing config in this file for testing purposes, so before I pushed the code I stashed this file. So it  doesn't work because those dependencies are indeed not present 🤦‍♂ I must've unstashed this file locally because I have them locally. It was so obvious that it would take me long time to actually check this.  Thank you again 🙏

Reply:
@mzgreen sorry if sounded like I was putting into questions your skills. I don't. This PR is not beginner level, quite obviously ;)

I am glad you could find how to fix the issue. Congrats for the green build.